docs/tutorial/tutorial-4.rst

```diff
@@ -0,0 +1,15 @@
+=====================================
+Let's draw on a canvas! (Gtk+ only for now)
+=====================================
+
+Explanation about Canvas here..
+
+.. image:: screenshots/tutorial-4.png
+
+Here's the source code
+
+.. literalinclude:: /tutorial/source/tutorial-4.py
+   :language: python
+
+
+In this example, we see a new Toga widget - :class:`.Canvas`.
```

Comment:
We can wait until the API has settled down, but we probably need a bit more detail than just "see the code". A little background on how contexts & drawing work would be good. Doesn't have to be *incredibly* detailed, but a few pointers would be good.

Reply:
The tutorial now has an explanation on how the widget works.